source/develop/release-management/features/external-network-provider.html.md

```diff
@@ -0,0 +1,402 @@
+---
+title: External Network Providers
+category: feature
+authors: mmirecki
+wiki_category: Feature
+wiki_title: Features/External Network Providers
+wiki_revision_count: 1
+wiki_last_updated: 2016-01-25
+feature_name: External Network Providers
+feature_modules: engine,vdsm
+feature_status: In Development
+---
+
+# External Network Providers
+
+
+## Summary
+
+Currently the Openstack Neutron integration allows oVirt to integrate with Openstack Neutron. This purpose of this feature is to simplify this to make it easier to add support for networks supplied by external providers. The providers will be able to provide networks using a simplified Neutron-like API.
```

Comment:
The first sentence is a tautology. How about saying like "Currently, with OpenStack Networking integration (add href), oVirt can use networks defined by OpenStack."

This purpose -> The purpose.

to simplify this *and* to make it